fr/contents.rst

```diff
@@ -76,8 +76,8 @@ Contenu
     :caption: Plugins & Packages
 
     standalone-packages
-    Authentication <https://book.cakephp.org/authentication/2/>
-    Authorization <https://book.cakephp.org/authorization/2/>
+    Authentification <https://book.cakephp.org/authentication/2/>
+    Autorisation <https://book.cakephp.org/authorization/2/>
```

Comment:
Up to you, but these should probably be the actual name of the plugin.

Reply:
Real question !
These are neither technical english terms commonly used in french (unlike "debug kit" for example), nor proper names (unlike "CakePHP"). Therefore, in french they sound as ugly as if the translator had simply forgotten to translate them.
One could translate "Plugin Authentication" ; it lets understand it as a proper name, but it's still odd, and the word "plugin" is redundant with category title and inconsistent with other plugins.
In my opinion, translate them makes the contents simply more meaningful.

Reply:
I think the idea is they will recognize it on packagist/etc. It's a name not an idea.

Reply:
But, the french translation is up to the french speakers.

Reply:
Doesn't sound like a name for french speakers. Then the question is : is there any place they need to identify these plugins by their original names ? Can't find in which case.
Anyway, you're the boss, boss ! Let me know.